charts/jindofsx/templates/fuse/daemonset.yaml

```diff
@@ -161,11 +161,21 @@ spec:
               subPath: hdfs-site.xml
           {{- end }}
           {{- end }}
-          {{- if .Values.secret }}
+          {{- if and .Values.secret .Values.UseStsToken }}
             - name: jindofs-secret-token
               mountPath: /token
               readOnly: true
           {{- end }}
+          {{- if .Values.secret }}
+            - name: jindofs-secret-token
+              mountPath: /token/AccessKeyId
+              subPath: {{ .Values.secretKey }}
+              readOnly: true
+            - name: jindofs-secret-token
+              mountPath: /token/AccessKeySecret
+              subPath: {{ .Values.secretValue }}
+              readOnly: true
```

Comment:
Is it required jindofs-secret-token/jindofs-secret-token  for fuse-only mode?

Reply:
But is it required for master-slave jindo cache?

Reply:
required